google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -118,7 +118,8 @@ protected void textChanged(DocumentEvent event) {
     );
 
     genAppYamlButton.addActionListener(new GenerateConfigActionListener(project, "app.yaml",
-        appEngineHelper::defaultAppYaml, appYaml, this::validateConfiguration));
+        () -> appEngineHelper.defaultAppYaml(FlexibleRuntime.java), appYaml,
```

Comment:
A decision needs to be made here: do we want to add a runtime picker to the facet as well, or always generate runtime: java app.yamls?